pkg/nodeipam/ipam/cloud_cidr_allocator_test.go

```diff
@@ -209,7 +208,7 @@ func TestUpdateNodeSubnetMaskSizes(t *testing.T) {
 			assert.NoError(t, err)
 
 			expectedVMSS := compute.VirtualMachineScaleSet{
-				Name: ptr.To("vmss"),
+				Name: lo.ToPtr("vmss"),
```

Comment:
Looks like some of the changes are using `github.com/Azure/azure-sdk-for-go/sdk/azcore/to`. Could you use the same one for consistency?